extensions/bodyparser-msgpack/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@loopback/bodyparser-msgpack",
+  "version": "0.0.1",
+  "description": "@loopback/bodyparser-msgpack",
+  "keywords": [
+    "loopback-extension",
+    "loopback"
+  ],
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "engines": {
+    "node": ">=10.16"
+  },
+  "scripts": {
+    "build": "lb-tsc",
+    "build:watch": "lb-tsc --watch",
+    "pretest": "npm run clean && npm run build",
+    "test": "lb-mocha \"dist/__tests__/**/*.js\"",
+    "clean": "lb-clean dist *.tsbuildinfo .eslintcache"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/strongloop/loopback-next.git",
+    "directory": "extensions/bodyparser-msgpack"
+  },
+  "author": "IBM Corp.",
+  "license": "MIT",
+  "files": [
+    "README.md",
+    "dist",
+    "src",
+    "!*/__tests__"
+  ],
+  "dependencies": {
+    "@loopback/core": "^2.9.4",
+    "@loopback/rest": "^6.1.0",
+    "msgpack-lite": "^0.1.26",
+    "tslib": "^2.0.0",
+    "type-is": "^1.6.18"
+  },
+  "devDependencies": {
+    "@loopback/build": "^6.2.1",
+    "@loopback/testlab": "^3.2.3",
+    "@types/msgpack-lite": "^0.1.7",
+    "@types/node": "^10.17.28",
+    "@types/type-is": "^1.6.3",
+    "typescript": "~3.9.7"
+  },
+  "copyright.owner": "IBM Corp.",
+  "publishConfig": {
+    "access": "public"
+  }
+}
```

Comment:
You are adding a new package now, did you follow our guide for [adding a new package]( https://github.com/strongloop/loopback-next/blob/4fbe6afb19f689b3a27aa4992377e3993ab62973/docs/site/DEVELOPING.md#adding-a-new-package)?